SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -371,9 +374,6 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	FSpatialFunctionalTestStepDefinition ClearSnapshotStepDefinition;
 
 private:
-	UPROPERTY(EditAnywhere, meta = (ClampMin = "0"), Category = "Spatial Functional Test")
-	int NumRequiredClients = 2;
-
 	// Number of servers that should be running in the world.
 	int NumExpectedServers = 0;
```

Comment:
Couldn't we remove this also?

Reply:
I think we still need NumExpectedServers as it's tested against the NumRegistered Servers in the test framework.